documentation/modules/rn-2.2.adoc

```diff
@@ -98,3 +98,7 @@ If a Plan CR references storage, network, or VMs by name instead of by ID, the r
 .Log archive file includes logs of a deleted migration plan or VM
 
 If you delete a migration plan and then run a new migration plan with the same name or if you delete a migrated VM and then remigrate the source VM, the log archive file created by the {project-short} web console might include the logs of the deleted migration plan or VM. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2023764[*BZ#2023764*])
+
+.If a target VM resource is deleted during the image conversion stage, the VM migration is still marked as successful.
+
+If you delete a target `VirtualMachine` CR during the 'Convert image to kubevirt' stage of the migration, the MTV Operator displays the state of the stage as `VirtualMachine CR not found`. However, the status of the VM migration appears as `Succeeded` in the `plan.yaml` file and in the web console. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2031529[*BZ#2031529*])
```

Comment:
> the MTV Operator displays the state of the stage as `VirtualMachine CR not found`

@mguetta1 How is it possible for the Operator to display the state of the stage? 
@sbeskin-redhat Is this a typo for the Migration details page of the web console?

> state of the stage

This is probably status, not state. Remove "of the stage" because you mention it in the subordinate clause.

> the status of the VM migration appears as `Succeeded`

Change to "The status of the migrated VM is `Succeeded`..." See the plan attachment in the bug for details.

> `plan.yaml` file 

Change to `Plan` CR. We do not call them yaml files in downstream docs.

Reply:
@apinnick I think the intention is the MTV UI
![mtv_ui_screenshot](https://user-images.githubusercontent.com/60384172/148955497-3442bb7c-473a-4a77-ae0c-d99f7812cca6.png)